app/src/main/java/com/example/smartshopper/services/FirebaseService.java

```diff
@@ -0,0 +1,61 @@
+package com.example.smartshopper.services;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import androidx.annotation.NonNull;
+import com.example.smartshopper.responseInterfaces.SetStrInterface;
+import com.example.smartshopper.responseInterfaces.UserInterface;
+import com.google.firebase.database.ChildEventListener;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.Query;
+import com.google.firebase.database.ValueEventListener;
+
+import java.util.HashSet;
+
+public class FirebaseService {
```

Comment:
would it be okay to move this to PlatformHelpers to avoid confusion? FirebaseService (from our A8) is replaced by PlatformHelpers in our project now.

Reply:
resolved